gulpfile.js

```diff
@@ -71,16 +77,50 @@ gulp.task('cleanup', function() {
   ]);
 });
 
+/**
```

Comment:
Is there a preference for range comments (`/* */ vs //`)?

Reply:
This file already seems to use `// comments`. Lets stick to the same convention.